gems/codders-dataset/OSVDB-108582.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
They are separate in OSVDB, though. MITRE treated postgresql and mysql the same. OSVDB split them into separate issues. I prefer we track all OSVDB and CVE IDs for Ruby issues so there's never a question as to whether we have something in our database. It would be nice if we had our own numbering system and then could attach CVEs and OSVDB IDs to those issues, but that's out-of-scope here.